core/routemgmt/common/apigw-utils.js

```diff
@@ -954,16 +950,14 @@ function makeResponseObject(resp, isWebAction) {
     return resp;
   }
 
-  var bodystr;
```

Comment:
not exactly related.. but simple update...
change line 953/940
`makeErrorResponseObject` -> `makeResponseObject`